tap_google_analytics/client.py

```diff
@@ -110,7 +110,12 @@ def _lookup_data_type(self, type, attribute, dimensions_ref, metrics_ref):
 
     @staticmethod
     def _generate_report_definition(report_def_raw):
-        report_definition = {"metrics": [], "dimensions": []}
+        report_definition = {
+            "metrics": [],
+            "dimensions": [],
+            "dimension_filters": [],
```

Comment:
@ajbosco I'm wondering if it might be more clear for users if we allow the top level key `dimensionFilterClauses` in the report definition. So the report definition is as close to the actual report request from https://developers.google.com/analytics/devguides/reporting/core/v4/basics as possible.

What do you think?